backend/adapter_processor/adapter_processor.py

```diff
@@ -97,6 +98,12 @@ def test_adapter(adapter_id: str, adapter_metadata: dict[str, Any]) -> bool:
             test_result: bool = adapter_instance.test_connection()
             logger.info(f"{adapter_id} test result: {test_result}")
             return test_result
+        # HACK: Remove after error is explicitly handled in VertexAI adapter
+        except json.JSONDecodeError:
```

Comment:
Do we get this exception directly originating from test connection of vertex_ai @chandrasekharan-zipstack ?

Reply:
Yes @gaya3-zipstack